kvui.py

```diff
@@ -817,6 +817,11 @@ def __init__(self, *args, **kwargs):
         boxlayout.add_widget(AutocompleteHintInput())
         self.add_widget(boxlayout)
 
+    def fix_heights(self):
+        for child in self.children:
+            if hasattr(child, "fix_heights"):
+                getattr(child, "fix_heights")()
```

Comment:
```suggestion
            fix_func = getattr(child, "fix_heights", None):
            if fix_func: fix_func()
```
This should be minutely faster to run, since this is run periodically, it might even be measurable. Definitaly not noticeable though.